generated/applicationinsights/log.out

```diff
@@ -125,97 +125,97 @@ warning | DeprecatedConfig | Using directive.56.suppress which is deprecated and
 warning | PreCheck/SchemaMissingType | The schema 'ComponentPurgeBody' with an undefined type and declared properties is a bit ambiguous. This has been auto-corrected to 'type:object'
-    - file:///Users/shenglol/Repos/azure-rest-api-specs/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json:567:5
+    - file:///home/jeskew/Workspaces/azure-rest-api-specs/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json:567:5
 warning | PreCheck/SchemaMissingType | The schema 'ComponentPurgeBodyFilters' with an undefined type and declared properties is a bit ambiguous. This has been auto-corrected to 'type:object'
-    - file:///Users/shenglol/Repos/azure-rest-api-specs/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json:587:5
+    - file:///home/jeskew/Workspaces/azure-rest-api-specs/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json:587:5
 warning | PreCheck/SchemaMissingType | The schema 'ComponentPurgeResponse' with an undefined type and declared properties is a bit ambiguous. This has been auto-corrected to 'type:object'
-    - file:///Users/shenglol/Repos/azure-rest-api-specs/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json:607:5
+    - file:///home/jeskew/Workspaces/azure-rest-api-specs/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json:607:5
 warning | PreCheck/SchemaMissingType | The schema 'ComponentPurgeStatusResponse' with an undefined type and declared properties is a bit ambiguous. This has been auto-corrected to 'type:object'
-    - file:///Users/shenglol/Repos/azure-rest-api-specs/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json:619:5
+    - file:///home/jeskew/Workspaces/azure-rest-api-specs/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json:619:5
 warning | PreCheck/CheckDuplicateSchemas | Checking for duplicate schemas, this could take a (long) while.  Run with --verbose for more detail.
 warning | Modeler/MissingType | The schema 'ComponentPurgeBodyFilters-value' has no type or format information whatsoever. Location:
-   file:///Users/shenglol/Repos/azure-rest-api-specs/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json#/components/schemas/ComponentPurgeBodyFilters-value
-warning | Skipping path '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Insights/components/{resourceName}/Annotations': Found mismatch between type segments (2) and name segments (1)
```

Comment:
These are presumably singleton children as well, though they must expose a `PUT` API if they were processed by the type generator in previous runs.

Reply:
This PR didn't make any changes to the `2015-05-01` types.md file, and the only changes to the types.json file were adding a `"Flags":0` property to all resource types. Not sure what caused this log alteration, but it appears to be noise.